src/pipdeptree/__init__.py

```diff
@@ -997,6 +997,15 @@ def get_parser():
             " ignore this argument."
         ),
     )
+    parser.add_argument(
+        "--without-unicode",
+        action="store_true",
+        default=False,
+        help=(
+            "Display dependency tree as text without using unicode characters."
+            "This option overrides default unicode detection."
+        ),
+    )
```

Comment:
I'd rather make this a flag of encoding; that the user can set to ascii if they don't want the utf characters 👍